state/validatorsInfoMap_test.go

```diff
@@ -0,0 +1,358 @@
+package state
+
+import (
+	"encoding/hex"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/stretchr/testify/require"
+)
+
+func TestShardValidatorsInfoMap_Add_Delete_Replace_SetValidatorsInShard_NilValidators(t *testing.T) {
```

Comment:
maybe stick to the rule: "one _ in test function names" ?
And use t.Run if more complex description is required

Reply:
Refactored name to `TestShardValidatorsInfoMap_OperationsWithNilValidators` and added t.Run for each nil operation